metecho/api/views.py

```diff
@@ -165,7 +165,11 @@ def feature_branches(self, request, pk=None):
         repo = gh.get_repo_info(
             request.user, repo_id=instance.get_repo_id(request.user)
         )
-        data = [branch.name for branch in repo.branches() if "__" not in branch.name]
+        data = [
+            branch.name
+            for branch in repo.branches()
+            if "__" not in branch.name and branch.name != repo.default_branch
+        ]
```

Comment:
@wlonk We also want to filter out branch names that are used by other (active, not-deleted) Metecho projects.